mkosi/__init__.py

```diff
@@ -66,6 +66,7 @@
 from .backend import (
     _FILE,
     ARG_DEBUG,
+    ArgsException,
```

Comment:
I'd name this `MkosiNotSupportedException` to make it very clear we only use this for unsupported combinations of options.